support-frontend/assets/pages/digital-subscription-checkout/components/endSummary/endSummary.jsx

```diff
@@ -4,31 +4,64 @@ import React from 'react';
 import * as styles from './endSummaryStyles';
 import { connect } from 'react-redux';
 import mapStateToProps from './endSummarySelector';
+import { getGiftOrderSummaryText } from '../helpers';
 
 export type EndSummaryProps = {
   priceDescription: string,
   promotion: string,
+  orderIsAGift: boolean,
+  digitalGiftBillingPeriod: 'Annual' | 'Quarterly',
+  price: string,
 }
 
 const Dot = () => <div css={styles.dot} />;
 
-function EndSummary({ promotion, priceDescription }: EndSummaryProps) {
+function EndSummary({
+  promotion, priceDescription, orderIsAGift = false, digitalGiftBillingPeriod, price,
+}: EndSummaryProps) {
+  const giftText = getGiftOrderSummaryText(digitalGiftBillingPeriod, price);
   return (
-    <ul css={styles.list}>
-      <li>
-        <Dot /><div css={styles.listMain}>{promotion}</div>
-        <span css={styles.subText}>{priceDescription}</span>
-      </li>
-      <li>
-        <Dot /><div css={styles.listMain}>14-day free trial</div>
-        <span css={styles.subText}>
-          Your first payment will occur after the trial ends
-        </span>
-      </li>
-      <li>
-        <Dot /><div css={styles.listMain}>You can cancel any time</div>
-      </li>
-    </ul>
+    <span>
+      {orderIsAGift ? (
+        <ul css={styles.list}>
```

Comment:
Maybe break these out into two subcomponents `GiftSummary` and `NonSummary`? I think it would be easier to read. You could also create a separate component for the individual list items to reduce code duplication.

Reply:
Sure thing - shall do 👍